Sources/MountebankSwift/Models/Config/Logs.Log+Codable.swift

```diff
@@ -24,7 +24,7 @@ extension Logs.Log {
         message = try container.decode(String.self, forKey: .message)
 
         if let dateString = try container.decodeIfPresent(String.self, forKey: .timestamp) {
-            timestamp = DateFormatter.shared.formatToDate(dateString)
+            timestamp = DateFormatter().formatToDate(dateString)
```

Comment:
Same. For 15+ it would be cheaper to use Date.ParseStrategy.parse. But caching the formatter would also work